velero-plugin-for-aws/object_store.go

```diff
@@ -208,6 +215,29 @@ func (o *ObjectStore) Init(config map[string]string) error {
 	return nil
 }
 
+// newSessionOptions creates a session.Options with the given config and profile. If
+// caCert and credentialsFile are provided, these will be used for the CustomCABundle
+// and the credentials for the session.
+func newSessionOptions(config aws.Config, profile string, caCert string, credentialsFile string) (session.Options, error) {
+	sessionOptions := session.Options{Config: config, Profile: profile}
+
+	if len(caCert) > 0 {
+		sessionOptions.CustomCABundle = strings.NewReader(caCert)
+	}
+
+	if credentialsFile != "" {
```

Comment:
nit: since caCert and credentialsFile both are string type, it'd be good to have the same checking way like checking `!= ""`.